core/players-handbook-2024/classes/class-cleric.xml

```diff
@@ -0,0 +1,447 @@
+		</sheet>
+	</element>
+
+	<element name="Potent Spellcasting" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_BLESSEDSTRIKES_POTENTSPELLCASTING">
+		<description>
+			<p class="indent"> <i>Potent Spellcasting</i>. Add your Wisdom modifier to the damage you deal with any Cleric cantrip.</p>
+		</description>
+		<sheet>
+			<description>You can add {{wisdom:modifier}} to the damage you deal with any Cleric cantrip.</description>
+			<description level="14">You can add {{wisdom:modifier}} to the damage you deal with any Cleric cantrip. Additionally, when you cast a Cleric cantrip and deal damage to a creature with it, you can grant yourself or another creature within 60 feet of you a number of Temporary Hit Points equal to {{doublewisdom:bonus}}. </description>
+		</sheet>
+		<rules>
+			<stat name="doublewisdom:bonus" value="wisdom:modifier" />
+			<stat name="doublewisdom:bonus" value="wisdom:modifier" />
+		</rules>
+	</element>
+
+	<element name="Level 10: Divine Intervention" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEINTERVENTION">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_CLERIC_DIVINE_INTERVENTION</requirements>
+		<description>
+			<p> You can call on your deity or pantheon to intervene on your behalf. As a Magic action, choose any Cleric spell of level 5 or lower that doesn't require a Reaction to cast. As part of the same action, you cast that spell without expending a spell slot or needing Material components. You can't use this feature again until you finish a Long Rest. </p>
+		</description>
+		<sheet alt="Divine Intervention" usage="1/Long Rest">
+			<description>As a Magic action, choose any Cleric spell of level 5 or lower that doesn 't require a Reaction to cast. As part of the same action, you cast that spell without expending a spell slot or needing Material components.</description>
+			<description level="20">As a Magic action, choose any Cleric spell of level 5 or lower that doesn 't require a Reaction to cast. As part of the same action, you cast that spell without expending a spell slot or needing Material components. 
+			You can also choose the <i>Wish</i> spell. If you do so, you can't use Divine Intervention again until you finish 2d4 Long Rests.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 14: Improved Blessed Strikes" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_IMPROVEDBLESSEDSTRIKES" >
+		<description>
+			<p>The option you chose for Blessed Strikes grows more powerful. </p>
+
+				<li><i>Divine Strike</i> The extra damage of your Divine Strike increases to 2d8.</li>
+				<li><i>Potent Spellcasting</i> When you cast a Cleric cantrip and deal damage to a creature with it, you can give vitality to yourself or another creature within 60 feet of yourself, granting a number of Temporary Hit Points equal to twice your Wisdom modifier.</li>
+
```

Comment:
Hmmm, think the presentation is a bit more defined as it is rather than indenting them and having them be plopped ontop of eachother like paragraphs

Reply:
The only reason I do indents now instead of lists is that the lists don't show up on the ability/feature cards for the pdf printout. I used to use them all the time till I realized they don't work with the card styling for some reason.